### ISSUE TITLE
[Bug] VisionTritonAttention drops MiMo-style window attention and sinks

### ISSUE THREAD

**Issue by qianyuandong45** [sgl-project/sglang#37983]
### Checklist

- [x] I searched related issues but found no solution.
- [x] The bug persists in the latest version.
- [x] Issues without environment info and a minimal reproducible demo are hard to resolve and may receive no feedback.
- [ ] If this is not a bug report but a general question, please start a discussion at https://github.com/sgl-project/sglang/discussions. Otherwise, it will be closed.
- [x] Please use English. Otherwise, it will be closed.

### Describe the bug

`VisionTritonAttention` does not propagate vision attention `window_size` and `s_aux` / attention sinks to the Triton prefill attention kernel.

For vision models whose ViT blocks rely on local-window attention and learned attention sinks, this silently changes the attention semantics when `triton_attn` is used as the multimodal/vision attention backend. The model can still run, but the visual features are computed with full attention and without sinks, which may lead to significantly degraded or incorrect multimodal outputs.

The issue is in the vision Triton path:

- `VisionTritonAttention.forward()` receives `**kwargs`, but does not pass `window_size` or `s_aux` to `context_attention_fwd`.
- `context_attention_fwd()` currently has no `window_size` or `sinks` arguments.
- The Triton `_fwd_kernel` only masks sequence bounds and optional causal constraints. It does not apply local-window constraints and does not include attention sinks in the online softmax denominator.

This is different from the FlashAttention3 vision path, which already supports `window_size` and sinks.

This was observed with a MiMo-V2.5-style vision model where most ViT blocks use local-window attention and learned sinks. Switching the multimodal attention backend to `triton_attn` caused incorrect visual understanding, while the FlashAttention path produced correct outputs.

### Reproduction

Reproduction requires a MiMo-style vision model whose ViT blocks use local-window attention and learned attention sinks.

Example launch pattern:

```bash
python3 -m sglang.launch_server \
  --model-path <model_path> \
  --trust-remote-code \
  --attention-backend triton \
  --mm-attention-backend triton_attn
```

Then send an image request to the OpenAI-compatible chat completion endpoint:

```python
import requests

payload = {
    "model": "<model_path>",
    "messages": [
        {
            "role": "user",
            "content": [
                {"type": "text", "text": "Describe this image."},
                {"type": "image_url", "image_url": {"url": "<image_url_or_data_url>"}},
            ],
        }
    ],
    "max_tokens": 128,
}

r = requests.post("http://localhost:30000/v1/chat/completions", json=payload)
print(r.status_code)
print(r.text)
```

Actual result:

The server runs, but the visual output can be incorrect because the Triton vision attention path computes full attention without attention sinks.

Expected result:

The Triton vision attention path should preserve the same `window_size` and sink semantics as the FlashAttention vision path.

### Environment

- SGLang version: main / v0.5.18
- Backend: `triton_attn` for multimodal vision attention
- GPU: NVIDIA B40 / Blackwell, also verified at kernel level on H20
- CUDA: CUDA 13.x environment on Blackwell; CUDA 12.x environment on H20
- Python: 3.12
- PyTorch: 2.8+ / 2.13 depending on the environment
- OS: Linux

**Comment by qianyuandong45** [comment]
Thanks for the PR. This matches what I observed with MiMo-V2.5.

For reference, I previously tested a similar local patch here:
https://github.com/qianyuandong45/sglang/commit/c7683ab361b1b7dd647a87bb12bc46d3e070ab38

With that patch, MiMo-V2.5 produced normal multimodal outputs on a Blackwell GPU environment using the Triton kernel backend. The core fix was to preserve `window_size` and `s_aux` / per-head sink semantics in the Triton vision attention path.

The key launch settings I used were:

```bash
python3 -m sglang.launch_server
--model-path <model_path>
--served-model-name mimo-v2.5
--pp-size 1
--dp-size 1
--tp-size 4
--trust-remote-code
--context-length 262144
--quantization fp8
--attention-backend triton
--mm-attention-backend triton_attn
```

The important part for this issue is `--mm-attention-backend triton_attn`. Other serving and deployment flags are not required to reproduce the attention-semantics issue.

One related note: I also noticed the same kwargs-forwarding gap in the FA4 vision path. My local patch forwards `window_size` and optional `s_aux` to `flash_attn_func(..., ver=4)`. Not sure whether this should be included here or handled as a separate follow-up PR, but it may be worth checking for MiMo-style models.
